apps/codelab/src/app/components/slides/closing-slide/codelab-closing-slide.component.spec.ts

```diff
@@ -1,13 +1,48 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { CodelabClosingSlideComponent } from './codelab-closing-slide.component';
+import { Router } from '@angular/router';
+import { SlidesDeckComponent } from '@codelab/slides/src/lib/deck/deck.component';
+import { MENU_ROUTES, MenuRoutes } from '../../../common';
 
 describe('CodelabClosingSlideComponent', () => {
   let component: CodelabClosingSlideComponent;
   let fixture: ComponentFixture<CodelabClosingSlideComponent>;
+  const routerStub = {
+    url: '/ngtest/currentlesson'
+  };
+
+  const slidesDeckComponentStub = {
+    previousLink: '',
+    nextLink: '',
+
+    setPrevious(link) {
+      this.previousLink = link;
```

Comment:
Instead of implementing this logic, just make setPrevious a spy and verify that it has been called